src/routes/index.js

```diff
@@ -1 +1,59 @@
 // ADD CODE HERE
+var express = require('express');
+const bodyParser = require('body-parser')
+const { getAllTodos, getCompletedTodos,completeOneTodo,addOneTodo } = require('./routes/index.js')
+const app = express()
+app.set('view engine', 'pug')
+app.use(express.static('public'))
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: true}))
+
+
+app.post('/complete', (request, response) => {
+  const id = request.body.id
```

Comment:
How are you planning to get the id from the body? I would think you would want the link or button that "completes" a todo to utilize the params in the url string. i.e. the route would be `/complete/:id` in the end